packages/react/dropdown-menu/src/DropdownMenu.stories.tsx

```diff
@@ -1569,6 +1570,43 @@ export const Chromatic = () => {
 };
 Chromatic.parameters = { chromatic: { disable: false } };
 
+export const InPopupWindow = () => {
```

Comment:
nit: can we move this just above the "Chromatic" story, as we like to keep this one last. Thanks!

Reply:
Done.